plugins/include/amxmodx.inc

```diff
@@ -159,6 +159,18 @@ forward client_connect(id);
  */
 forward client_connectex(id, const name[], const ip[], reason[128]);
 
+/**
+ * Called after the client's admin flags are changed.
+ *
+ * @param id        Client index
+ * @param oldflags  Old client flags
+ * @param newflags  New client flags
+ * @param set       Flag set id
+ *
+ * @noreturn
+ */
+forward client_flags_updated(id, oldflags, newflags, set);
```

Comment:
Maybe use same terminology that is used in `set_user_flags`?
Client ID is `index`
Set ID is just `id`